public/app/features/dashboard/state/PanelModel.ts

```diff
@@ -369,6 +369,11 @@ export class PanelModel implements DataConfigSource, IPanelModel {
     const version = getPluginVersion(plugin);
 
     if (plugin.onPanelMigration) {
+      // hack to clear saved plugins for v9-pre
```

Comment:
I don't think this is necessary, for the one dashboard I had built to test this, it was.  Real dashboards should not have `pluginVersion` saved in them.